numba/core/types/misc.py

```diff
@@ -141,17 +141,27 @@ def key(self):
 class CPointer(Type):
     """
     Type class for pointers to other types.
+
+    Attributes
+    ----------
+        dtype : The pointee type
+        addrspace : [int]
```

Comment:
```suggestion
        addrspace : int
```
the square bracket from the example (`frozenset[int]`) is for generics only

Reply:
Fixed.